src/tabbar.cpp

```diff
@@ -427,6 +427,16 @@ void TabBar::paintEvent(QPaintEvent *e)
 
         QRect tabTextRect = style()->subElementRect(QStyle::SE_TabBarTabText, &tabopt, this);
 
+#if QT_VERSION < QT_VERSION_CHECK(6, 0, 0)
+        // Add Padding to left, right. Padding is 4px. Add 5 to account for 
+        // Extra pixel from border.
+        tabTextRect.setX(tabTextRect.x() + 5);
+        tabTextRect.setWidth(tabTextRect.width() - 5);
+#else
+        // Qt6 correctly adds left and right padding but now incorrectly adds
+        // top or bottom padding.
+        tabTextRect.setY(tabTextRect.y() - 4);
+#endif
```

Comment:
Magic numbers are evil. Please introduce a local constant (`const int padding = 4)` with a comment linking it to the value of `QTabBar::tab::padding` in `resources/css/style.css` (also add a sibling comment in `resources/css/style.css`).

Reply:
Done